staging/src/k8s.io/api/admissionregistration/v1/types.go

```diff
@@ -307,6 +307,18 @@ type ValidatingWebhook struct {
 	// include any versions known to the API Server, calls to the webhook will fail
 	// and be subject to the failure policy.
 	AdmissionReviewVersions []string `json:"admissionReviewVersions" protobuf:"bytes,8,rep,name=admissionReviewVersions"`
+
+	// MatchConditions is a list of conditions on the AdmissionRequest ('request') that must be met
```

Comment:
nit: what does `('request')` refer to? I think you can drop it.

Reply:
IMO the CEL-specific documentation should go on the `Expression` field. I think the plan there is to have something pretty minimal, and just reference https://github.com/kubernetes/website/pull/39642 (in merged form)